src/engine/source/hlp/src/specificParsers.cpp

```diff
@@ -183,6 +210,43 @@ bool configureIgnoreParser(Parser& parser, std::vector<std::string_view> const&
     return true;
 }
 
+bool configureJsonParser(Parser& parser, std::vector<std::string_view> const& args)
+{
+    // If no arguments parse object
+    if (args.empty())
+    {
+        parser.options.push_back("object");
+    }
+    else
+    {
+        // If one argument, it must be a valid json type or "any"
+        if (args.size() == 1)
+        {
+            if (validJsonType(std::string {args[0]}))
```

Comment:
If the map is a string_view it is not necessary to copy the memory and create a temporary value (which will be downloaded after its single use) from a string_view.

Reply:
got it